packages/react/src/components/PageHeader/PageHeader.tsx

```diff
@@ -241,6 +235,176 @@ PageHeaderContent.propTypes = {
   pageActions: PropTypes.node,
 };
 
+/**
+ * ----------------
+ * PageHeaderContentPageActions
+ * ----------------
+ */
+interface PageHeaderContentPageActionsProps {
+  /**
+   * Provide child elements to be rendered inside PageHeaderContentPageActions.
+   */
+  children?: React.ReactNode;
+  /**
+   * Specify an optional className to be added to your PageHeaderContentPageActions
+   */
+  className?: string;
+  /**
+   *  The PageHeaderContent's page actions collabpsible Menu button label
+   */
+  menuButtonLabel?: string;
+  /**
+   * The PageHeaderContent's page actions
+   */
+  pageActions?: React.ReactNode;
+}
+const PageHeaderContentPageActions = React.forwardRef<
+  HTMLDivElement,
+  PageHeaderContentPageActionsProps
+>(
+  ({
+    className,
+    children,
+    menuButtonLabel = 'Actions',
+    pageActions,
+    ...other
+  }: PageHeaderContentPageActionsProps) => {
```

Comment:
I think this can be simplified more:

1. `children` is unused.
2. `other` is unused.
3. `PageHeaderContentPageActionsProps` is inferred from the generics on `forwardRef`.

Also, if `children` is unused, should it be removed from the component's interface, or is the intention that it will be used in the future?

```suggestion
  ({
    className,
    menuButtonLabel = 'Actions',
    pageActions,
  }) => {
```

Reply:
We'll most likely change the way the page actions are consumed, instead of having them passed in through the `pageActions` prop I would like to have them be passed in through `children` so I'll keep that in for now. There will be another issue opened to address that